Makefile

```diff
@@ -239,6 +239,8 @@ promote-staging-manifest:
 	@cp -r manifest_staging/deploy .
 	@rm -rf charts
 	@cp -r manifest_staging/charts .
+	@helm package ./charts/gatekeeper -d ./charts/gatekeeper
```

Comment:
Won't the tgz files get wiped out by `rm -rf charts`

Reply:
it does, i mentioned `Since it gets wiped out every release, we'll only be hosting latest chart` in the description

Reply:
oh sorry I missed that. do we want to also keep the old charts?